Resources/views/frontend/checkout/change_payment.tpl

```diff
@@ -1,5 +1,15 @@
 {extends file="parent:frontend/checkout/change_payment.tpl"}
 
+{* Hidden input for retrieving the id of Mollie's Apple Pay payment mean *}
+{block name='frontend_checkout_payment_content'}
```

Comment:
why is this important? please add comments, i think it might break something if removed?